imap_processing/ialirt/l0/process_swapi.py

```diff
@@ -202,19 +203,21 @@ def process_swapi_ialirt(
 
     # Extract energy values from the calibration lookup table file
     calibration_lut_table["timestamp"] = pd.to_datetime(
-        calibration_lut_table["timestamp"], format="%m/%d/%Y %H:%M"
-    )
-    calibration_lut_table["timestamp"] = calibration_lut_table["timestamp"].to_numpy(
-        dtype="datetime64[ns]"
+        calibration_lut_table["timestamp"]
     )
+    subset_sweep = calibration_lut_table[
+        calibration_lut_table["Sweep #"] == SWEEP_NUMBER
```

Comment:
I don't think this is right. We want the sweep id that is coming down with the ialirt packet, not fixed on 2 because we are now on sweep id 4 and 5.